cloud/deployment/fromfile/fromfile.go

```diff
@@ -411,36 +625,36 @@ func deploymentExists(existingDeployments []astro.Deployment, deploymentNameToCr
 
 // deploymentFromName takes existingDeployments and deploymentName as its arguments.
 // It returns the existing deployment that matches deploymentName.
-func deploymentFromName(existingDeployments []astro.Deployment, deploymentName string) astro.Deployment {
+func deploymentFromName(existingDeployments []astroplatformcore.Deployment, deploymentName string) astroplatformcore.Deployment {
 	for i := range existingDeployments {
-		if existingDeployments[i].Label == deploymentName {
+		if existingDeployments[i].Name == deploymentName {
 			// deployment that matched name
 			return existingDeployments[i]
 		}
 	}
-	return astro.Deployment{}
+	return astroplatformcore.Deployment{}
 }
 
 // getClusterInfoFromName takes clusterName and org as its arguments.
 // It returns the clusterID and list of nodepools if the cluster is found in the organization.
 // It returns an errClusterNotFound if the cluster does not exist in the organization.
-func getClusterInfoFromName(clusterName, org string, coreClient astrocore.CoreClient) (string, []astrocore.NodePool, error) {
+func getClusterInfoFromName(clusterName, organizationShortName string, platformCoreClient astroplatformcore.CoreClient) (string, []astroplatformcore.NodePool, error) {
 	var (
-		existingClusters []astrocore.Cluster
+		existingClusters []astroplatformcore.Cluster
 		err              error
 	)
 
-	existingClusters, err = organization.ListClusters(org, coreClient)
+	existingClusters, err = organization.ListClusters(organizationShortName, platformCoreClient)
 	if err != nil {
 		return "", nil, err
 	}
 
 	for _, cluster := range existingClusters { //nolint
 		if cluster.Name == clusterName {
-			return cluster.Id, cluster.NodePools, nil
+			return cluster.Id, *cluster.NodePools, nil
```

Comment:
shouldn't it be like this since cluster.NodePools is `[]` and not `*[]`
```suggestion
			return cluster.Id, cluster.NodePools, nil
```

Reply:
nvm its correct